tests/main/views/test_reset_password.py

```diff
@@ -262,8 +262,16 @@ def test_token_created_before_last_updated_password_cannot_be_used(self):
         assert reset_password.EXPIRED_PASSWORD_RESET_TOKEN_MESSAGE in error_elements[0].text_content()
         assert self.data_api_client.update_user_password.called is False
 
+    @pytest.mark.parametrize("user_role", (
+        "admin",
```

Comment:
Might as well add the other admin roles in as well (`admin-ccs-category` etc)?

Reply:
:heavy_check_mark: